sbi/neural_nets/ratio_estimators.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
+
+import torch
+from torch import Tensor, nn
+
+
+class RatioEstimator(nn.Module, ABC):
```

Comment:
just a note for #1072 : this should inherit from `Estimator` and adopt shape conventions from #1066 
no need to change it as part of this PR.

Reply:
okay! but for now I'll just move on?

Reply:
yes, no need to fix this now.